3/T4/index.js

```diff
@@ -0,0 +1,59 @@
+const min = document.getElementById('min');
+const max = document.getElementById('max');
+const buttonFindFriendlyNumbers = document.getElementById('find');
+
+buttonFindFriendlyNumbers.addEventListener('click',() =>{
+    if(+min.value < +max.value){
+        if(+min.value >0&&+max.value >0){
+            findFriendlyNumbers(+min.value,+max.value);
+        }else{
+            alert('The numbers should be > 0');
+            min.value = '';
+            max.value = '';
+        }
+    }else{
+        alert('The beginning of the range cannot be smaller than the end');
+        min.value = '';
+        max.value = '';
```

Comment:
maybe it was entered by mistake. In this case you can change values from min to max and from max to min :)

Reply:
changed